src/containers/AccountModal/forms/EditPassword.tsx

```diff
@@ -11,7 +11,11 @@ import { useAccountStore } from '#src/stores/AccountStore';
 import { changePasswordWithOldPassword, changePasswordWithToken, logout } from '#src/stores/AccountController';
 import useQueryParam from '#src/hooks/useQueryParam';
 
-const ResetPassword: React.FC = () => {
+type Prop = {
```

Comment:
can we handle the type inline

Reply:
66f9c2d3a1f608c3acb984248ddd3140419439da